src/index.js

```diff
@@ -57,9 +65,16 @@ class EllipisWithTooltip extends React.Component {
 }
 
 EllipisWithTooltip.propTypes = {
-    placement: PropTypes.string,
-    children: PropTypes.node.isRequired,
-    style: PropTypes.object,
+  placement: PropTypes.string,
+  children: PropTypes.node.isRequired,
+  style: PropTypes.object,
+  delayShow: PropTypes.number,
+  delayHide: PropTypes.number,
+};
+
+EllipisWithTooltip.defaultProps = {
+  delayHide: 0,
```

Comment:
giving an undefined value wouldn't be better? don't undefined props  stop from spreading?